idp/authmiddleware/authmiddleware.go

```diff
@@ -0,0 +1,202 @@
+// Package authmiddleware contains a http middleware for the authentication with the IdentityProvider-App
+package authmiddleware
+
+import (
```

Comment:
Run goimports over imports. (Grouped imports by standard and external packages)

Reply:
Done